src/main/java/hw_5/Task1/MyCity/Building.java

```diff
@@ -0,0 +1,18 @@
+package hw_5.Task1.MyCity;
+
+import hw_5.Task1.MyClass.Person;
+
+public class Building {
+    public Person[] persons;
```

Comment:
Do we need this line?

Reply:
fixed

Reply:
not fixed yet
![Screenshot_2](https://github.com/user-attachments/assets/cc4f214b-dce8-43e6-b83e-ec6a10d035a3)

Reply:
fixed